libmscore/sig.cpp

```diff
@@ -411,12 +411,12 @@ void TimeSigMap::read(XmlReader& e, int fileDivision)
 //   SigEvent
 //---------------------------------------------------------
 
-SigEvent::SigEvent(const SigEvent& e)
-{
-    _timesig = e._timesig;
-    _nominal = e._nominal;
-    _bar     = e._bar;
-}
+//SigEvent::SigEvent(const SigEvent& e)
+//{
+//    _timesig = e._timesig;
+//    _nominal = e._nominal;
+//    _bar     = e._bar;
+//}
```

Comment:
why uncommenting this here but removing the others?

Reply:
That was a mistake. Will fix now.